keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/verification/ContainerTestVerification.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.titus.verification
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup.Location
+
+data class ContainerTestVerification(
+  val repository: String,
+  val tag: String = "latest",
+  val location: Location
+) : Verification {
+  override val type: String = "container-tests"
+  override val id: String = "$repository/$tag"
+}
```

Comment:
This is the API class that will appear in delivery config. I wonder if we could merge this with the `ContainerJobConfig` class? I wanted to keep it as simple as possible though. I haven't (yet) added the capability to run digests but that would be easy enough.

Reply:
Merging with `ContainerJobConfig` sounds reasonable to me.

As for digests, I'm tempted to not expose support for them in the delivery config until users come to us with an explicit use case.

Reply:
+1 to Lorin's suggestion.